### DISCUSSION TITLE
why can't I read this compiled core?

### DISCUSSION THREAD

**Discussion by daxinhub** [EmulatorJS/EmulatorJS#862] [General]
Friends, why can't I read this compiled core? Use the official website's method to build it; FBNEO cannot run KoF (games like Neogeo cannot run, so I want to build my own Neogeo core)

**Comment by daxinhub** [comment]
![image](https://github.com/user-attachments/assets/7468d5b1-ab7e-4b97-9dbe-aebf773dd4d8)
![image](https://github.com/user-attachments/assets/59bb121e-f4ee-48b6-a41a-812d89ee126e)
![image](https://github.com/user-attachments/assets/be64036b-9a83-4cb7-aba5-210ce4c7b047)

**Reply by ethanaobrien** [reply]
You need to use this branch of retroarch, and enable EJS_DEBUG_XX. if that doesn’t work, I honestly don’t know what you’re doing

https://github.com/EmulatorJS/RetroArch

**Reply by daxinhub** [reply]
> 您需要使用 retroarch 的这个分支，并启用 EJS_DEBUG_XX。如果这不起作用，老实说，我不知道你在做什么
> 
> https://github.com/EmulatorJS/RetroArch

I am using this compiled core, and I just want to build some cores myself, such as the fbalpha2012-wasm.data core, which is not included in the emulator JS. And, I built the core using emulatorjs/RetroArch, but I put the built fbalpha2012-wasm.data into/ EmulatorJS/data/cores， However, the emulator JS failed to read

**Reply by michael-j-green** [reply]
Not sure if you've seen it, but use this repo to do the builds: https://github.com/EmulatorJS/build

**Reply by daxinhub** [reply]
Thank you, I suggested trying this, but there are still some parameters that I can't understand at the moment

**Reply by michael-j-green** [reply]
For your efforts compiling, use the sections in the cores.json file in the build repo to specify what you want to build (I suggest creating a whole new cores.json file with only the core you want to build in it).

The README.MD file for that repo contains descriptions of each JSON attribute.

**Reply by daxinhub** [reply]
> For your efforts compiling, use the sections in the cores.json file in the build repo to specify what you want to build (I suggest creating a whole new cores.json file with only the core you want to build in it).
> 
> 
> 
> The README.MD file for that repo contains descriptions of each JSON attribute.

🤣，I don't know where the error occurred, I can't find the compiled core, and there's no logs output; When I run bash build. sh again, it prompts build. sh: line 114: jq: command not found

![image](https://github.com/user-attachments/assets/5a04bf3f-fd64-4b2c-adbf-866ae8d84c9c)
![image](https://github.com/user-attachments/assets/5612128d-2879-4e6f-896a-0bf4b4a2a323)

**Reply by daxinhub** [reply]
> For your efforts compiling, use the sections in the cores.json file in the build repo to specify what you want to build (I suggest creating a whole new cores.json file with only the core you want to build in it).
> 
> 
> 
> The README.MD file for that repo contains descriptions of each JSON attribute.

It can compile on Ubuntu？

**Reply by michael-j-green** [reply]
Ok, thanks for that, we're missing the install command for jq. You'll need to run ``apt install jq``. I'll push an update to the build_env.sh script later to include that.

Logs are sent to ./output

**Reply by daxinhub** [reply]
> Ok, thanks for that, we're missing the install command for jq. You'll need to run `apt install jq`. I'll push an update to the build_env.sh script later to include that.
> 
> Logs are sent to ./output

Thank you, I'll give it a try

**Reply by daxinhub** [reply]
> Ok, thanks for that, we're missing the install command for jq. You'll need to run ``apt install jq``. I'll push an update to the build_env.sh script later to include that.
> 
> 
> 
> Logs are sent to ./output

Haha🤣, I can't find Wasm. It seems there's an error. Emsdk is in the build directory, right？

![image](https://github.com/user-attachments/assets/82c2e553-0a28-4284-90b4-2a6c124d8f0c)

**Reply by daxinhub** [reply]
![image](https://github.com/user-attachments/assets/b016c580-8b38-4d22-9d34-2110eb490207)

**Reply by daxinhub** [reply]
> Ok, thanks for that, we're missing the install command for jq. You'll need to run `apt install jq`. I'll push an update to the build_env.sh script later to include that.
> 
> Logs are sent to ./output

I have built it, but it still cannot be read；Still making mistakes like the first two pictures.Can't fbalpha2012 run?

**Comment by daxinhub** [comment]
Solved,. net requires MIME
